### DISCUSSION TITLE
Lung Nodule Detection

### DISCUSSION THREAD

**Discussion by Asma-Alkhaldi** [Project-MONAI/MONAI#5263] [Q&A]
I'm trying to run (Detection) code .. I have downloaded LUNA16 dataset and I modified the sections that need for modification in (luna16_prepare_env_files.py) to prepare the data as you did, But when I run it to do the resampling I got this error:
ValueError: Data list file /home/workspace/LUNA16/original/dataset_fold0.json does not exist.

I think you prepared the data to be read from JSON file, and I need to have a look at (dataset_fold0.json) so I can prepare the downloaded data as you did

**Comment by Can-Zhao** [comment]
> /home/workspace/LUNA16/original/dataset_fold0.json

Thanks for reaching out. Have you checked whether `/home/workspace/LUNA16/original/dataset_fold0.json` exist?

**Reply by Can-Zhao** [reply]
The json files can be downloaded as:
[tutorial prepare data split json](https://github.com/Project-MONAI/tutorials/tree/9987da22c680c96565071ef20f8c1132620faad2/detection#:~:text=We%20follow%20the,in%20world%20coordinate.)

**Reply by Asma-Alkhaldi** [reply]
(dataset_fold0.json) is not exist and I don't know how to generate it until I see an example. The link you provided above is downloading the JSON environment only, and inside the environment, there is an attribute called (data_list_file_path) which has the path of dataset_fold0.json. So, can you provide me an example of (dataset_fold0.json) so I can prepare mine?

**Reply by Can-Zhao** [reply]
No worries. Have you clicked the link https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/LUNA16_datasplit-20220615T233840Z-001.zip in the tutorial? It should download dataset_fold0.json
